### DISCUSSION TITLE
Overhang angle adjustment seems to be missing

### DISCUSSION THREAD

**Discussion by Zephyrus67** [supermerill/SuperSlicer#3684] [General]
I have just been printing a printhead for my V6 hotend for my Voron 2.4 Stealthburner.
It was sliced with Cura but I had some issues with overhangs causing curling/warping after 25-30 layers. This is in the top layers...and not about first layer adhesion issues.

I found out after testing with various temperatures for both bed and hotend that temperatures alone would not do the trick so I started looking at the overhang parameters and I found out I could control the angle of when something was considered an overhang..and that saved me...I thought....because I could not define how much cooling I wanted per overhangs...that is a lack in Cura...

So..I found superslicer..and looked through it all and I was amazed..."wow..what a great slicer..I dont need Cura anymore"...

I saw that here I was actually able to set how much cooling I want on bridge/overhangs.

I was happy..but then I realized that overhang angles seems to be fixed in this app...no way to adjust when something is considered an overhang by angle...

If this feature was in Superslicer it would be my new slicer but I see no real reason to swap for now.

This is really a lack of feature I dont understand.

**Comment by neophyl** [comment]
Not by angle no but you can adjust it.  Under Print Settings>Perimeter & Shell>Overhangs you can adjust the thresholds for Bridge Speed/Fan and Flow.  By setting the perimeter overlap percentage (or fixed value) you can adjust what it treats as a bridge/overhang.

**Reply by Zephyrus67** [reply]
I have played with thta but not seen a way to get a part in my print seen as overhang that I definitely think SHOULD be. With angle it is easy.
